frontend/public/components/factory/ListPage/ListPageCreate.tsx

```diff
@@ -97,9 +98,10 @@ const ListPageCreate: React.FC<ListPageCreateProps> = ({
   groupVersionKind,
   children,
 }) => {
-  const kind = typeof groupVersionKind !== 'string' ? groupVersionKind.kind : groupVersionKind;
+  const kind =
+    typeof groupVersionKind !== 'string' ? getReference(groupVersionKind) : groupVersionKind;
```

Comment:
`getReference` handles the single kind string, so this can be simply

```suggestion
  const reference = getReference(groupVersionKind);
```